examples/.env.example

```
@@ -1,5 +1,7 @@
 # MCP plugin -- used by the MCP adapter to expose routes as tools
-JWT_SECRET=change-me
+JWT_ISSUER=change-me
+JWT_AUDIENCE=https://idp.example.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_AUDIENCE key should go before the JWT_ISSUER key

(UnorderedKey)